test/tests/test_harness/csvdiff_diff_gold.i

```diff
@@ -0,0 +1,59 @@
+[Mesh]
```

Comment:
I'm not sure you need to duplicate these files either. I think the right way to test this is to just soft link everything you need or just re-use existing test inputs but specify the different test harness options like you are doing.

Reply:
yeah, much better with soft links

Reply:
why are these in the test folder to begin with? To do the diff tests you have to modify
```
test/tests/test_harness/exodiff.i
test/tests/test_harness/diff
python/TestHarness/test_Diff.py
python/TestHarness/tests
```
Seems un-moosian :)